src/Cafe/CMakeLists.txt

```
@@ -1,10 +1,16 @@
 project(CemuCafe)
 
 include_directories(".")
+
+if((CMAKE_C_COMPILER_ID MATCHES "GNU") OR (CMAKE_C_COMPILER_ID MATCHES "Clang"))
```

Use `STREQUAL` instead of `MATCHES`. The latter is for regular expressions.

```suggestion
if((CMAKE_C_COMPILER_ID STREQUAL "GNU") OR (CMAKE_C_COMPILER_ID STREQUAL "Clang"))
```

---

Yep, thanks. Fixed.